docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Develop a node.js “Hello World” application, using the devfile specification. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to be on a cluster. To create a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube start].
+* To access your namespaces, download link:https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/[kubectl].
+* To execute the devfile specification, install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/cli_tools/developer-cli-odo#installing-odo[odo cli tool].
+
+.Procedure
+
+. Build your Kubernetes cluster by running `minikube start`.
+. When you use minikube for the first time, you must enable Ingress in order to access your devfile projects. Enable Ingress by running `minikube addons enable ingress`.
```

Comment:
```suggestion
. When you use minikube for the first time, you must enable Ingress to access your devfile projects. Enable Ingress by running `minikube addons enable ingress`.
```
- Per IBM SG, avoid "in order to". Use "to" in most cases to avoid wordiness.
- To me, this step feels more like a prerequisite step. Maybe move this as a Note admonition for the minikube item in the `Prerequistes` section?